eos/b-decays/observables.cc

```diff
@@ -1600,208 +1599,267 @@ namespace eos
             R"(The option "form-factors" selects the form factor parametrization.)",
             {
                 // B_s -> K^* l nubar
-                make_observable("B_s->K^*lnu::dBR/ds", R"(d\mathcal{B}(B_s\to \bar{K}^*\ell^-\bar\nu)/dq^2)",
+                make_observable("B_s->K^*lnu::dBR/dq2", R"(d\mathcal{B}(B_s\to \bar{K}^*\ell^-\bar\nu)/dq^2)",
                         Unit::InverseGeV2(),
-                        &BsToKstarLeptonNeutrino::differential_branching_ratio,
-                        std::make_tuple("q2")),
-
-                make_observable("B_s->K^*lnu::A_FB(q2)", R"(A_\mathrm{FB}(B_s\to \bar{K}^*\ell^-\bar\nu)(q^2))",
-                        Unit::None(),
-                        &BsToKstarLeptonNeutrino::differential_forward_backward_asymmetry,
-                        std::make_tuple("q2")),
+                        &BToVectorLeptonNeutrino::differential_branching_ratio,
+                        std::make_tuple("q2"),
+                        Options{ { "U", "u" }, {"q", "s"} }),
```

Comment:
We also need `{"I", "1/2"}`.
I also realize that it is missing in the `B_s -> D_s^* l nu`, can you fix that in a separate commit (with "0" instead of "1/2")?